BetterVanillaWorldGen/RockUnderHellException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Terraria.Localization;
+
+namespace AdvancedWorldGen.BetterVanillaWorldGen
+{
+	public class RockUnderHellException : Exception
```

Comment:
Are you sure about the need of an empty exception ?

Reply:
Slightly improves clarity when used, compared to just using the base Exception